src/components/labels/Label.jsx

```diff
@@ -77,6 +77,7 @@ export const LABEL_COLORS_SET = {
   PEACH: 'peach',
   MUSTARD: 'mustard',
   GRAY: 'gray',
+  ARCHROMATIC: 'achromatic',
```

Comment:
typo :)

Reply:
maybe Michal wanted `aromatic` colors 😄

Reply:
omg :D

Reply:
I recommend vscode plugin 'code spell checker'

Reply:
thx! I'll try it

Reply:
Or maybe our tests should catch that... hm